core/src/main/java/org/jruby/javasupport/util/JavaClassConfiguration.java

```diff
@@ -124,4 +126,18 @@ public synchronized void include(String name) {
         included.add(name);
         excluded.remove(name);
     }
+    
+    public static class AdvancedFieldConfiguration {
```

Comment:
I don't like the vagueness of "Advanced" since we might have something more "advanced" in the future. Perhaps "DirectFieldConfiguration" would be more descriptive?